pkg/volume/csi/csi_plugin.go

```diff
@@ -109,7 +109,7 @@ func (h *RegistrationHandler) ValidatePlugin(pluginName string, endpoint string,
 }
 
 // RegisterPlugin is called when a plugin can be registered
-func (h *RegistrationHandler) RegisterPlugin(pluginName string, endpoint string, versions []string) error {
+func (h *RegistrationHandler) RegisterPlugin(pluginName string, endpoint string, versions []string, pluginClientTimeout time.Duration) error {
```

Comment:
Would it make sense to use `pluginClientTimeout` on the line 133 below?

Reply:
yes it would be better